spec/features/Welcome/index_spec.rb

```diff
@@ -10,4 +10,17 @@
 
     expect(current_path).to eq("/quotes")
   end
+
+  it 'redirects to /quotes after searching keyword and returns the total number of quotes found, the search param and the top 10 quotes from results with the author/category' do
+    visit root_path
+
+    fill_in :keyword, with: 'I am'
+
+    click_on "Search for Quotes"
+    expect(page).to have_content("165 quotes found")#total count of quotes found
+    expect(page).to have_content("I am") #showing query search
+    expect(page).to have_content("Top 10")#might need to rewrite better/somewhow include or count quote objects
+    expect(page).to have_content("Rabbi Hillel") #rigid test, I have to figure out a way to be more flexible with the results
```

Comment:
You could try using fixtures for your specs so you always know what the content will be. That way you could be confident that the content won't change and lead to test failures.

Reply:
This part I still struggle a little bit with, does that mean something along the lines of `expect(<something>).to eq(String)` ? I struggle with what the <something> is, or how to define that given item/object? unless this is where we have divs to specify sections of the page or something?